Application/mobile/app_code/lib/l10n/app_en.arb

```diff
@@ -102,6 +102,9 @@
   "errorOccurredDuringSettingProfile": "An error occurred while setting up the profile.",
   "errorOccurredDuringSettingStripe": "An error occurred while setting up Stripe.",
   "errorOccurredDuringUpdateUserInformation": "An error occurred while updating user information.",
+  "faq": "Questions",
+  "faqAnswer": "FAQ",
+  "faqTitle": "Hi {name},\nhow can we\nhelp you?",
```

Comment:
I don't think '\n' should be in translation, should be a width cap in the user interface instead of '\n'. And it's looking a bit weird in the app